packages/api/src/event/medical/canvas-event-listener.ts

```diff
@@ -0,0 +1,51 @@
+import CanvasSDK from "@metriport/core/external/canvas/index";
+import { createFullNote } from "@metriport/core/external/canvas/note";
+import { getEnvVarOrFail } from "@metriport/shared";
+import { PatientEvents, patientEvents, PatientEvent } from "./patient-event";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+import { Util } from "../../shared/util";
+
+const log = Util.log(`[CANVAS EVENT LISTENER]`);
+
+export default function () {
+  log(`Setting up listener`);
+  patientEvents().on(PatientEvents.CANVAS_INTEGRATION, async (event: PatientEvent) => {
+    log(`Received event: ${JSON.stringify(event, null, 2)}`);
```

Comment:
left-over? if we want to keep the log, let's not format the json, please

Reply:
I want to keep the log. Removing formatting